src/DIPS.Xamarin.UI/Controls/Popup/PopupLayout.xaml.cs

```diff
@@ -0,0 +1,124 @@
+﻿using System;
+using DIPS.Xamarin.UI.Extensions;
+using Xamarin.Forms;
+using Xamarin.Forms.Xaml;
+
+namespace DIPS.Xamarin.UI.Controls.Popup
+{
+    [ContentProperty(nameof(MainContent))]
+    [XamlCompilation(XamlCompilationOptions.Compile)]
+    public partial class PopupLayout : ContentView
```

Comment:
Remember to add comments to all public properties, our build somehow does not go red if we forget anymore :(